workload.go

```diff
@@ -62,32 +76,59 @@ func (w WorkloadConfig) Config() WorkloadConfig {
 }
 
 func NewWorkloadConfigWithDefaults(c WorkloadConfig) WorkloadConfig {
-	if c.RateControl.OuterLoopRate == 0 {
-		c.RateControl.OuterLoopRate = 50
+	if c.Name == "" {
+		panic("must specify workload Name")
 	}
 
-	if c.Visualization.LatencyHistMin == 0 {
-		c.Visualization.LatencyHistMin = 0
+	if c.WorkloadScale <= 0 || c.WorkloadScale >= 1 {
```

Comment:
Shouldn't this be `<` and `>`? As it stand you can't have a workload that's exactly 0% or 100%? Am I reading this right?

Reply:
I'm using WorkloadScale == 0 as the default I'd expect to see if they don't specify it (so we wouldn't want to scale).  As for 1.0, I was going to allow it, then decided it's just making us do pointless scaling math and possibly indicates an unintentional user error.  These are the kind of things that will make a lot more sense if I write it all in a comment as you suggested.